docs/protected-handlers.md

```diff
@@ -65,3 +68,30 @@ fetch("/api/protected", {
 ```
 
 If you want to read more about `appBridgeState`, check [App Bridge](./app-bridge.md) documentation.
+
+### Using `useAuthenticatedFetch()` hook
+
+Instead of manually attaching headers with AppBridge context, you can use `useAuthenticatedFetch()` hook
+
+Since it requires AppBridge, it's only available in browser context. It depends on `Window` object,
+so your app will break if Next.js tries to render it server-side. Hence, ensure component that uses the hook is imported with dynamic()
+
+Component must be within `AppBridgeProvider` to have access to the AppBridge
+
+```tsx
+import { useAuthenticatedFetch } from "@saleor/app-sdk/app-bridge";
+import { useEffect } from "react";
+
+export const ClientComponent = () => {
+  const fetch = useAuthenticatedFetch();
+
+  useEffect(() => {
```

Comment:
Yes, http is an effect, you can't run `fetch` outside of it, because it will re-run on every render. I think its covered in React docs quite well - not sure if we should explain it, its quite far abstraction from our use case IMO

Reply:
Right, no need to duplicate the react docs :+1: